src/pwa.js

```diff
@@ -1,38 +0,0 @@
-let hasInteracted, shouldReload;
```

Comment:
It's possible I'm missing something, but I don't see much of a reason to keep this as a PWA, at least now without altering our set up quite a bit. I don't think we're getting much out of it being a PWA at the moment.

Reply:
+1 on dropping PWA mode. It was a cool tech demo, but I'm skeptical that our PWA is even used these days.